adafruit_ov7670.py

```diff
@@ -671,60 +679,60 @@ def _set_flip(self):
         self._write_register(_OV7670_REG_MVFP, mvfp)
 
     @property
-    def flip_x(self):
+    def flip_x(self) -> bool:
         """Get or set the X-flip flag"""
         return self._flip_x
 
     @flip_x.setter
-    def flip_x(self, value):
+    def flip_x(self, value: int) -> None:
```

Comment:
I think `value` can be `bool` instead of `int`.

Pythons truthiness logic for numbers would apply if a number were passed, but boolean are the values it's expecting.